crane: panic: runtime error: index out of range [0] with length 0

### Describe the bug

We (w/@dentrax) encountered an error while working with rebasing option.

We typed `crane rebase` without providing any argument and got the following error:

```shell=
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/google/go-containerregistry/cmd/crane/cmd.NewCmdRebase.func1(0xc000286f00?, {0x1862e60?, 0x0?, 0x0?})
        github.com/google/go-containerregistry/cmd/crane/cmd/rebase.go:41 +0xfe6
github.com/spf13/cobra.(*Command).execute(0xc000286f00, {0x1862e60, 0x0, 0x0})
        github.com/spf13/cobra@v1.4.0/command.go:856 +0x67c
github.com/spf13/cobra.(*Command).ExecuteC(0xc000280000)
        github.com/spf13/cobra@v1.4.0/command.go:974 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.4.0/command.go:902
github.com/spf13/cobra.(*Command).ExecuteContext(...)
        github.com/spf13/cobra@v1.4.0/command.go:895
main.main()
        github.com/google/go-containerregistry/cmd/crane/main.go:34 +0xce
```

### To Reproduce

just type `crane rebase` without giving any arguments.

### Expected behavior

What we expected from this is displaying the help message without throwing an error.

### Additional context

Add any other context about the problem here.

- Output of `crane version`: 0.9.0
- Registry used (e.g., GCR, ECR, Quay)

--- a/cmd/crane/cmd/rebase.go
+++ b/cmd/crane/cmd/rebase.go
@@ -35,7 +35,7 @@ func NewCmdRebase(options *[]crane.Option) *cobra.Command {
 	rebaseCmd := &cobra.Command{
 		Use:   "rebase",
 		Short: "Rebase an image onto a new base image",
-		Args:  cobra.MaximumNArgs(1),
+		Args:  cobra.MinimumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if orig == "" {
 				orig = args[0]

